Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,338 @@
+                            break;
+
+                    case 3: SellerRegistration(sellerList); 
+                            break;
+
+                    case 4: ProductRegistration(productsList); 
+                            break;
+
+                    case 5:                
+                            {
+                                int opcion = 0;
+                                int answer;                            
+                                bool boolAnswer = true;
+                                while (opcion != 5)
+                                {
+                                    do
+                                    {
+                                        Console.WriteLine("--------------------");
+                                        Console.WriteLine("Reports Menu");
+                                        Console.WriteLine("1. Total Sales");
+                                        Console.WriteLine("2. Customers older to 55 years.");
+                                        Console.WriteLine("3. Customers who meet years in this month.");
+                                        Console.WriteLine("4. Products low stock.");
+                                        Console.WriteLine("5. Return.");
+                                        Console.WriteLine("Select an option");
+                                        boolAnswer = Int32.TryParse(Console.ReadLine(),out answer);
+                                    }   
+                                        while (boolAnswer == false);                              
+                                    opcion = answer;
+                                    switch (opcion)
+                                    {
+                                        case 1: SalesReport(salesList);
+                                                break; 
+                                                
+                                        case 2: CustomersReport(customersList);
+                                                break;
+                                                
+                                        case 3: CustomerReportAge(customersList);
+                                                break; 
+                                                
+                                        case 4: ProductsReport(productsList);
+                                                break; 
+                                                
+                                        case 5: break;
+                                    }
+                                }
+                            }
+                            break;
+                }
+            }
+            Console.WriteLine("Thank for your visit.");
+        }
+
+        public static void SalesReport(List<Sale> salesList)
+        {
+            bool p = true;
+            Console.WriteLine("Total day sales :"+ salesList.Count);

```

Comment:
When you have to concat strings you don't have to use "+" the reason is because it's very inefficient and the best way is $"Total day sales : {salesList.Count}"